skema/text_reading/text_reading/Dockerfile

```diff
@@ -38,31 +38,44 @@ RUN sbt publishLocal
 # =============================================================================
 # CREATE A PYTHON VENV AND UPGRADE PYTHON TOOLS
 # =============================================================================
-ENV VIRTUAL_ENV=/opt/automates_venv
-RUN python3 -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+#ENV VIRTUAL_ENV=/opt/automates_venv
+#RUN python3 -m venv $VIRTUAL_ENV
+#ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+#
+#RUN pip install --upgrade setuptools
+#RUN pip install wheel
+# =============================================================================
+
+# =============================================================================
+# SETUP THE SKEMA TR ENVIRONMENT
+# =============================================================================
+RUN mkdir -p /skema/text_reading
+COPY . /skema/text_reading
+WORKDIR /skema/text_reading
+# Compile the fat jar
+RUN sbt assembly
```

Comment:
This will work, of course, especially if the machine building the image does not have sbt installed.  Running assembly in the container tends to be very slow, in my experience, and means that sbt and Scala are required in the image as well.

Reply:
Yes, it is slow, but it is just at build time. We will be posting the image to docker hub. 

The idea is that by having xDD creating the container from our image, they already have the jar ready and can just run the shell script without having to compile or run sbt.